webapp/gulp/config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   server: {
     settings: {
       root: dest,
-      host: 'localhost',
+      host: '0.0.0.0',
```

Comment:
I'd prefer localhost as the default here so people don't accidentally expose stuff